File: rails-demo/app/controllers/complete_ui_controller.rb
class CompleteUiController < ApplicationController
  def base
    @title = 'Base'
    render layout: "complete-ui"
  end

  def charts
    @title = 'Charts'
    render layout: "complete-ui"
  end

  def plugins
    @title = 'Plugins'
    render layout: "complete-ui"
  end

end

File: laravel-demo/app/Http/Controllers/CompleteUiController.php
<?php

namespace App\Http\Controllers;

use App\User;
use App\Http\Controllers\Controller;

class CompleteUiController extends Controller
{
    public function base()
    {
        return view('complete-ui.base', ['title' => 'Base']);
    }

    public function charts()
    {
        return view('complete-ui.charts', ['title' => 'Charts']);
    }

    public function plugins()
    {
        return view('complete-ui.plugins', ['title' => 'Plugins']);
    }

}
